content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -447,7 +469,10 @@ After you [determine the storage location](#determine-the-storage-address) for y
 - If you plan to configure CoreWeave storage for the team, contact [support](mailto:support@wandb.com) to verify that your bucket is configured correctly in CoreWeave and to validate your team's configuration, since the storage details cannot be changed after the team is created.
 {{% /alert %}}
 
-<!-- (Commented out until MT is supported) 1. If you are configuring CoreWeave storage for Multi-tenant Cloud, switch to the browser window where you previously began to create the new team to find the W&B organization ID previously. Otherwise, l -->
+Select your deployment type to continue.
+
+{{< tabpane text=true >}}
+{{< tab header="Dedicated Cloud / Self-Hosted" value="dedicated" >}}
 
 1. If you’re connecting to a cloud-native storage bucket in another cloud or to an S3-compatible storage bucket like MinIO for team-level BYOB in your Dedicated cloud or Self-managed instance, you **must** add the bucket path to the `GORILLA_SUPPORTED_FILE_STORES` environment variable and then restart W&B, before following the rest of these steps to use the storage bucket for a team.
```

Comment:
doesn't seem like this is properly formatted in the preview?
<img width="928" height="727" alt="Screenshot 2025-07-10 at 17 03 30" src="https://github.com/user-attachments/assets/229b705a-1681-4405-bff0-578a24cd522d" />